steer_0: 300
speed_0: 305
steer_range: 40
speed_range: 20